app/controllers/application_controller.rb

```diff
@@ -18,10 +18,14 @@ def user_matches_url
   end
 
   def logout
-    session[:user_id] = nil
+    cookies.delete(:auth_token)
   end
 
-  def login(user_id)
-    session[:user_id] = user_id
+  def login(user,remember_me=false)
```

Comment:
Space missing after comma.<br>Surrounding space missing in default value assignment.